lib/xboxkrnl/xboxkrnl.h

```diff
@@ -2111,6 +2080,8 @@ XBAPI VOID NTAPI RtlInitializeCriticalSection
     IN PRTL_CRITICAL_SECTION CriticalSection
 );
 
+#define RtlDeleteCriticalSection(CriticalSection) ((void)CriticalSection)
```

Comment:
This macro still feels dangerous for when `CriticalSection` is some address math (unlikely, but also shows the problem of macros).
Not sure if an `static`-function is much better though.